docs/code_examples/systemd.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Start Errbot chatbot
+After=network.service
+
+[Service]
+Type=forking
+Environment="LC_ALL=en_US.UTF-8"
+ExecStart=/path/to/errbot/virtualenv/bin/errbot --daemon --config /path/to/errbot/config.py
+WorkingDirectory=/path/to/errbot/
+User=errbot
```

Comment:
We should add `Restart=on-failure` (or perhaps even `always`) to ensure systemd restarts the bot should it crash for any reason.